docs/providers/aws/guide/variables.md

```diff
@@ -50,6 +50,7 @@ You can define your own variable syntax (regex) if it conflicts with CloudFormat
 - [Properties exported from Javascript files (sync or async)](#reference-variables-in-javascript-files)
 - [Pseudo Parameters Reference](#pseudo-parameters-reference)
 - [Read String Variable Values as Boolean Values](#read-string-variable-values-as-boolean-values)
+- [Merge pseudo-variable](#merge-pseudo-variable)
```

Comment:
Let's name it: _Merge multiple object/array sources_ ("Pseudo variable" term is used with some AWS variables, but we don't use this term for Serverless Variables)